packages/flame/lib/src/parallax.dart

```diff
@@ -93,13 +95,17 @@ abstract class ParallaxRenderer {
   /// How to fill the screen with the image, always proportionally scaled.
   final LayerFill fill;
 
+  final FilterQuality filterQuality;
+
   ParallaxRenderer({
     ImageRepeat? repeat,
     Alignment? alignment,
     LayerFill? fill,
+    FilterQuality? filterQuality,
   })  : repeat = repeat ?? ImageRepeat.repeatX,
         alignment = alignment ?? Alignment.bottomLeft,
-        fill = fill ?? LayerFill.height;
+        fill = fill ?? LayerFill.height,
+        filterQuality = filterQuality ?? FilterQuality.low;
```

Comment:
Maybe we should default to `none` or `high` instead?

Reply:
For me, high was not appropriate, as seems to be just a more detailed version of scaling, i.e. the pixelated appearance is still lost.

I was looking for the nearest neighbour scaling, since I want to maintain the pixelated appearance. 

I just left it as low as that's what the existing flutter method defaults to. I guess this hits the right balance for performance/appearance for most needs. 

From the comments on the method:

///  * `filterQuality`: Use this to change the quality when scaling an image.
///     Use the [FilterQuality.low] quality setting to scale the image, which corresponds to
///     bilinear interpolation, rather than the default [FilterQuality.none] which corresponds
///     to nearest-neighbor.

Reply:
if the default value is `.low`, then we should keep that, otherwise this would become a breaking change.